lib/gitlab/ci/templates/pipeline/GradleJavaPipeline.yml

```diff
@@ -42,6 +49,9 @@ stages:
 
 workflow:
   rules:
+    - if: '$SAFE_TEST =~ /true/i'
```

Comment:
This is a clever approach. We'll definitely want to introduce more variables using this pattern in future iterations

Reply:
Thanks!